04.examples/news-classification/scripts/evaluate/Dockerfile

```
@@ -0,0 +1,26 @@
+FROM python:3.10 AS builder
+
+WORKDIR /work
+
+COPY evaluate-requirements.txt . 
+RUN python -m venv venv && \
+    venv/bin/pip install --no-cache-dir -r evaluate-requirements.txt
+
+COPY . .
+
+FROM python:3.10
+
+WORKDIR /work
+
+COPY --from=builder /work /work
+
+ENV VIRTUAL_ENV=/work/venv
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+ENV PYTHONUNBUFFERED=1
+
+RUN deepeval set-local-model --model-name=llama3.2 \
+    --base-url="http://ollama:11434/v1/" \
+    --api-key="ollama"
+
+ENTRYPOINT [ "python", "-u", "evaluate.py" ]
+CMD [ "--config-file", "/configs/evaluate-config.json", "--model-path", "in/model"]
```

```suggestion
CMD [ "--config-file", "/configs/evaluate-config.json", "--model-path", "/in/model"]
```

I am doubtful that current container can find the fine tuned model.

Plan yaml declares for the model to be mounted on `/in/*`.
But here, `WORKDIR` points `/work` so current directory of the container is `/work`.
This command, `python -u evaluate.py ... --model-path in/model`, may looks `/work/in/model`, as far as I understand.

Is this intended?